addon/ng2/tasks/serve-webpack.ts

```diff
@@ -15,7 +15,8 @@ module.exports = Task.extend({
     let lastHash = null;
     let webpackCompiler: any;
 
-    var config: NgCliWebpackConfig = new NgCliWebpackConfig(this.project, commandOptions.target, commandOptions.environment).config;
+    var config: NgCliWebpackConfig = new NgCliWebpackConfig(this.project, commandOptions.target, commandOptions.environment, commandOptions.outputPath).config;
```

Comment:
After some research, I saw that setting up `contentBase` in 'webpack-dev-server' configuration does not bundle the files into the desired folder. So i did use this plugin [write-file-webpack-plugin](https://github.com/gajus/write-file-webpack-plugin) which forces 'webpack-dev-server' to write a bundle folder.